src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -502,14 +502,14 @@ public static LocalDateTime parseLocalDateTime(String dateTimeString, String pat
      * Parses a `LocalDateTime` object from separated date, hour and minute strings.
      *
      * <p>required parameter format:
-     * date: dd/MM/yyyy  hour: H   min:m
+     * date: EEE, dd MMM, yyyy  hour: H   min:m
      * Example: If date is 01/04/2014, hour is 23, min is 59.
      */
     public static LocalDateTime parseLocalDateTime(String date, String hour, String min) {
```

Comment:
This method needs to be renamed to something more descriptive since it is now for a specific use case. Maybe something like `parseLocalDateTimeForSessionsForm` to match `formatDateForSessionsForm`.